pandas/core/window/rolling.py

```diff
@@ -1351,8 +1352,21 @@ def apply_func(values, begin, end, min_periods, raw=raw):
 
         return apply_func
 
-    def sum(self, *args, **kwargs):
+    def sum(self, *args, engine=None, engine_kwargs=None, **kwargs):
         nv.validate_window_func("sum", args, kwargs)
+        if maybe_use_numba(engine):
+            if self.method == "table":
```

Comment:
might be worth a common function for this?

Reply:
If I inline the `args` assignment it would essentially be a 1 line function.

Also I anticipate numba adding an `axis` argument incrementally so I would imagine some methods needing the `if self.method == "table":` and others not.